pulse/loaders/confluence.py

```diff
@@ -0,0 +1,45 @@
+"""Contains confluence loaders for loading data from confluence into the vectorstore."""
+import logging
+import sys
+
+from atlassian import Confluence
+from lxml import etree
+from pulse.services import confluence
+from pulse.vectorstore import vectorstore
+
+logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
+logger = logging.getLogger("pulse.loaders.confluence")
+
+def load_all_labelled_docs():
+    """
+    Load all documents from a confluence space with "pulse" label into the vector store
+    """
+    # rest call to confluence with the username and api key
```

Comment:
could also just ingest everthing, problem is if there's too much html/markup stuff it doesn't really parse properly

Reply:
maybe we don't care